### ISSUE TITLE
wb-ready.wb not firing in Edge

### ISSUE THREAD

**Issue by NoisyMonk** [wet-boew/wet-boew#8268]
Hello,

Are there any known issues with WET-BOEW and Edge?

For some reason, my multi-step application works fine as long as you don't hit refresh on any page in Edge. As you navigate the site everything is fine, but as soon as you refresh any page, the "wb-ready.wb" event never fires, so none of my code in "$(document).on("wb-ready.wb", function () { ...code... });" get executed.

Unfortunately I'm stuck using WET 4.0.24, so even if I could just get confirmation that this was a known issue back in .24, that would also satisfy my needs as well.

**Comment by duboisp** [comment]
> Are there any known issues with WET-BOEW and Edge?

Not that I am aware that is about refreshing the page and executing wet-boew. 

The next step will be to create a test example that reproduce your issue with a fresh copy of WET 4.0.24 and check if it can be reproduced with the latest version of WET.

**Comment by NoisyMonk** [comment]
Yes that's the plan. Unfortunately it's been difficult to get a Windows 10 device on our internal network to test with. Hopefully soon.

I'll update this issue when I can. Not much more to be done until then.

Thanks!

**Comment by LaurentGoderre** [comment]
@NoisyMonk you can use saucelabs to test browsers you don't have on your machine. You can even use it to test you local site.

https://saucelabs.com/

**Comment by NoisyMonk** [comment]
Thanks @LaurentGoderre, unfortunately my organization rarely allows 3rd party software, but I'll look into it!

More on my issue; I can also reproduce this in IE11 - although it happens significantly less often; perhaps only 10% of the time in IE, whereas it's more like 80% of the time in Edge.

Furthermore, I can also reproduce with 4.0.27. Does the "$(document).on("wb-ready.wb", function () { ...code... });"  need to be loaded in a specific place, like before or after wet.js is loaded?

**Comment by LaurentGoderre** [comment]
The order shouldn't matter unless it is itself in an event handler

**Comment by duboisp** [comment]
@NoisyMonk 

In fact it's after or before the jQuery.js and not wet.js

* **After:** You can use a plain ```<script>...code...<script>``` tag
* **Before:** You would need to specify the defer flag like ```<script defer="defer">...code...<script>``` (That option might not work on every browser)

**Comment by NoisyMonk** [comment]
Yes, definitely loading the script after jQuery.js. Just wanted to make sure about wet.js.
